docs/rest_api.md

```diff
@@ -416,6 +416,31 @@ Authorization requirements:
 Power cycle the node named `<node>`, and set it's next boot device to
 PXE. If the node is powered off, this turns it on.
 
+#### node_set_bootdev
+
+`POST /node/<node>/dev/<dev>/set_bootdev`
+
+Sets the nodes next boot device persistently
+
+device <dev> can be set to:
+
+  none  : Do not change boot device order
```

Comment:
This doesn't seem like a coherent option; if you were going to us this, why make the API call at all?